tests/ios/MASVS-CODE/MASTG-TEST-0085.md

```diff
@@ -117,3 +117,4 @@ The `list_bundles` command lists all of the application’s bundles that are not
 ```bash
 ...itudehacks.DVIAswiftv2.develop on (iPhone: 13.2.3) [usb] # ios bundles list_bundles
 Executable    Bundle                                       Version  Path
+```
```

Comment:
Please go back to v1.5.0 and get the missing text from the original test in 
 Document/0x06i-Testing-Code-Quality-and-Build-Settings.md